torch/_dynamo/variables/torch.py

```diff
@@ -236,15 +236,20 @@ def call_function(
         unspec_python_args = check_unspec_python_args(args, kwargs)
         options = VariableTracker.propagate(self, args, kwargs.values())
 
+        def get_full_name(value):
+            if hasattr(value, "__module__") and hasattr(value, "__name__"):
+                return value.__module__ + "." + value.__name__
+            return None
+
         if self.value is torch._functorch.vmap.vmap_impl:
             return TorchHigherOrderOperatorVariable.make(
                 self.value,
                 source=self.source,
             ).call_function(tx, args, kwargs)
-        elif self.value in config.constant_functions:
+        elif (full_name := get_full_name(self.value)) in config.constant_functions:
```

Comment:
This was to make the config more determinstic. Please advise if this is ok @voznesenskym @ezyang